src/main/java/htsjdk/variant/vcf/VCFCompoundHeaderLine.java

```diff
@@ -202,6 +202,9 @@ protected VCFCompoundHeaderLine(String line, VCFHeaderVersion version, Supported
     }
 
     private void validate() {
+        if (countType == VCFHeaderLineCount.INTEGER && count <= 0)
+            throw new IllegalArgumentException(String.format("Invalid count number, with fixed count the number should be 1 or higher: key=%s name=%s type=%s desc=%s lineType=%s count=%s",
+                    super.getKey(), name, type, description, lineType, count));
```

Comment:
`super.` is unnecessary here as `getKey()` is not overridden

Reply:
can remove it, just did the same as 1 line down ;)